### ISSUE TITLE
Eureka Tests fail with Java 9

### ISSUE THREAD

**Issue by chaitan64arun** [ewolff/microservice#20]
Any new members using Java 9 to test this project, you might encounter test failures like below.

```catalogLoads(com.ewolff.microservice.eurekaserver.ApplicationTests)  Time elapsed: 0.515 sec  <<< ERROR!
org.springframework.web.client.HttpClientErrorException: 404 null
	at org.springframework.web.client.DefaultResponseErrorHandler.handleError(DefaultResponseErrorHandler.java:91)
	at org.springframework.web.client.RestTemplate.handleResponse(RestTemplate.java:700)
	at org.springframework.web.client.RestTemplate.doExecute(RestTemplate.java:653)
	at org.springframework.web.client.RestTemplate.execute(RestTemplate.java:613)
	at org.springframework.web.client.RestTemplate.getForEntity(RestTemplate.java:312)
	at com.ewolff.microservice.eurekaserver.ApplicationTests.catalogLoads(ApplicationTests.java:25)
```
The root cause is `java.lang.TypeNotPresentException: Type javax.xml.bind.JAXBContext not present`.
Refer Java 9 explanation[here](http://mail.openjdk.java.net/pipermail/jdk9-dev/2016-May/004309.html)

So just down grade your JDK to work until there is a fix available.

**Comment by ewolff** [comment]
Thanks for pointing this out!

**Comment by emazzotta** [comment]
This fixed it for me: https://stackoverflow.com/a/48136912/2596187

**Comment by ewolff** [comment]
Thanks! I fixed it with 5a74df2f1620e9998adf8c741c0059b8f5569bdd .
